.travis.yml

```diff
@@ -22,7 +23,4 @@ script:
   - docker logs docker_zabbix-agent-modbus_1
   - docker exec docker_zabbix-agent-modbus_1 sh -c "zabbix_get -s localhost -k agent.ping"
   - docker exec docker_zabbix-agent-modbus_1 sh -c "zabbix_get -s localhost -k modbus_read[172.16.238.2:5020,1,2,3,i,1,1]"
-  - python tests/test_modbus.py
-  - python tests/test_modbus_errors.py
-  - python tests/test_modbus_64bit.py
-  - python tests/test_modbus_connection_string.py
+  - pytest -v
```

Comment:
This:
> Add new command in .travis.yml to run new file on commit.
won't be needed [here](https://github.com/v-zhuravlev/libzbxmodbus/tree/master/tests#to-add-new-tests) any more, right?

Reply:
yepp 4. is not needed anymore